src/utils/navigation.ts

```diff
@@ -239,6 +239,11 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.UPDATE_BENEFICIAL_OWNER_TYPE_URL,
     nextPage: [config.UPDATE_BENEFICIAL_OWNER_TYPE_URL]
   },
+  [config.TRUST_DETAILS_URL]: {
+    currentPage: config.TRUST_DETAILS_PAGE,
+    previousPage: () => config.TRUST_INTERRUPT_PAGE,
+    nextPage: [config.TRUST_ENTRY_URL, config.ID, config.TRUST_INVOLVED_URL]
+  },
```

Comment:
Up to you if you want to leave this in now but it's currently under review(meeting tomorrow) so may need to be changed again